docker-compose.yaml

```diff
@@ -5,6 +5,37 @@ services:
       target: production
     image: ffc-pay-dps
     container_name: ffc-pay-dps
+    depends_on:
+      - ffc-pay-dps-postgres
+      - ffc-pay-dps-azurite
     environment:
       APPINSIGHTS_CLOUDROLE: ffc-pay-dps-local
       APPINSIGHTS_CONNECTIONSTRING: ${APPINSIGHTS_CONNECTIONSTRING}
+      AZURE_STORAGE_USE_CONNECTION_STRING: "true"
+      AZURE_STORAGE_CREATE_CONTAINERS: "true"
+      POLLING_INTERVAL: ${POLLING_INTERVAL:-10000}
+      MESSAGE_QUEUE_HOST: ${MESSAGE_QUEUE_HOST}
+      MESSAGE_QUEUE_PASSWORD: ${MESSAGE_QUEUE_PASSWORD}
+      MESSAGE_QUEUE_USER: ${MESSAGE_QUEUE_USER}
+      FILESEND_TOPIC_ADDRESS: ${SEND_TOPIC_ADDRESS:-ffc-pay-file-send}${MESSAGE_QUEUE_SUFFIX}
+      POSTGRES_DB: ffc_pay_dps
+      POSTGRES_HOST: ${POSTGRES_HOST:-ffc-pay-dps-postgres}
+      POSTGRES_PASSWORD: ${POSTGRES_SCHEMA_PASSWORD:-ppp}
+      POSTGRES_PORT: 5432
+      POSTGRES_SCHEMA_NAME: ${POSTGRES_SCHEMA_NAME:-public}
+      POSTGRES_USERNAME: ${POSTGRES_SCHEMA_USERNAME:-postgres}
+      AZURE_STORAGE_ACCOUNT_NAME: ${AZURE_STORAGE_ACCOUNT_NAME:-devstoreaccount1}
+      AZURE_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-dps-azurite:10000/devstoreaccount1;
+
+  ffc-pay-dps-postgres:
+    image: postgres:11.4-alpine
+    environment:
+      POSTGRES_DB: ffc_pay_dps
+      POSTGRES_PASSWORD: ppp
+      POSTGRES_USERNAME: postgres
+
+  ffc-pay-dps-azurite:
```

Comment:
I think we should set this to `ffc-pay-batch-azurite`, then for local development it will automagically share storage with `ffc-pay-gateway`, `ffc-pay-batch-processor` etc so all can be run together.

I think changing any reference to `ffc-pay-dps-azurite` to `ffc-pay-batch-azurite` should magically make that work.